docker/docker-compose.yml

```diff
@@ -19,17 +19,20 @@ services:
             start_period: 40s
         # Only exposing /healthcheck endpoint
         labels:
-            - "traefik.backend=oms-mailer"
+            - "traefik.backend=mailer"
             - "traefik.port=4000"
-            - "traefik.frontend.rule=Path:/services/oms-mailer/api/healthcheck;PathPrefixStrip:/services/oms-mailer/api"
+            - "traefik.old.frontend.rule=Path:/services/oms-mailer/api/healthcheck;PathPrefixStrip:/services/oms-mailer/api"
+            - "traefik.new.frontend.rule=Path:/api/mailer/healthcheck;PathPrefixStrip:/api/mailer"
+            - "traefik.old.frontend.priority=110"
+            - "traefik.new.frontend.priority=110"
             - "traefik.frontend.priority=110"
```

Comment:
Will Traefik use this priority? Since the priority for both the old and the new have also been specified

Reply:
Here, the priority doesn't matter as these endpoints have different prefixes and are not overlapping each other. Main thing is that these should override the frontend priorities, so requests like this won't be sent to frontend.